view/GameView/HUDScreen.py

```diff
@@ -153,6 +158,9 @@ def send_action_pressed(self) -> None:
             # reset gadget / property selection
             self.__selected_gad_prop_idx = None
 
+        if ret:
+            self._update_active_char(active=False)
```

Comment:
kosmetisch: Charakter wechselt während einem Zug ständig zwischen aktiv und nicht aktiv

evtl. kritisch: timing issues möglich
Wenn ret gesetz wurde ist die Nachricht bereits an den Server geschickt worden. Es ist also theoretisch möglich, dass `getActiveCharacter()` in `_update_active_char(...)` bereits den nächsten aktiven Spieler enthält.

Lösung: Variable im Client, die letzten aktiv markierten Charakter enthält (damit könnte auch die kosmetische Anmerkung gefixt werden)

Reply:
kritisch ist nicht so schlimm, da bei GameStatus active "zurückgesetzt" wird